src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/Stylus/Common/StylusPlugin.cs

```diff
@@ -92,17 +92,34 @@ internal void RawStylusInput(RawStylusInput rawStylusInput)
             // Only fire if plugin is enabled and hooked up to plugincollection.
             if (__enabled && _pic != null)
             {
-                switch (rawStylusInput.Report.Actions)
+                try
                 {
-                    case RawStylusActions.Down:
-                        OnStylusDown(rawStylusInput);
-                        break;
-                    case RawStylusActions.Move:
-                        OnStylusMove(rawStylusInput);
-                        break;
-                    case RawStylusActions.Up:
-                        OnStylusUp(rawStylusInput);
-                        break;
+                    switch (rawStylusInput.Report.Actions)
+                    {
+                        case RawStylusActions.Down:
+                            OnStylusDown(rawStylusInput);
+                            break;
+                        case RawStylusActions.Move:
+                            OnStylusMove(rawStylusInput);
+                            break;
+                        case RawStylusActions.Up:
+                            OnStylusUp(rawStylusInput);
+                            break;
+                    }
+                }
+                catch (Exception e)
+                {
+                    // this code running in Stylus Input thread
```

Comment:
i think this can be worded a little more clearly:


>This code is running on the Stylus Input thread and has the chance to call out into app code. If the app code throws an exception that is not caught, then the app will crash. We don't want to ignore all exceptions, so we catch the exception and dispatch it to the main thread, allowing developers to handle it inside the handler for the `Application.DispatcherUnhandledException` event.

Does that sound right?

Reply:
Thanks @stevenbrix . I will change it.